content/docs/reactivesearch/react-searchbox/searchbase.md

```diff
@@ -0,0 +1,113 @@
+---
+title: 'SearchBase API Reference'
+meta_title: 'API Reference for SearchBase'
+meta_description: 'SearchBase is a lightweight and platform agnostic search library for Elasticsearch.'
```

Comment:
Wrong meta description. It's a component. We went over this distinction in the last call.